api/v1alpha4/ibmpowervscluster_types.go

```diff
@@ -36,6 +36,7 @@ type IBMPowerVSClusterSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// ServiceInstanceID is the id of the power cloud instance where the vsi instance will get deployed
+	// +kubebuilder:validation:MinLength=1
```

Comment:
lets not change change anything in the older version, will focus only on v1beta1 and onwards,